### PR TITLE
Correctly handle the Krakensbane and floating origin, unpacked part insertion

### PR DESCRIPTION
**Pull request by eggrobin**
3rd point of the 10-point plan.

### CODE REVIEW

File: ksp_plugin/plugin.cpp [hunk 1]
```diff
@@ -418,9 +418,8 @@ void Plugin::InsertOrKeepLoadedPart(
     }
   } else {
     Instant const previous_time = current_time_ - Δt;
-    enum class LocalTag { tag };
-    using MainBodyCentred =
-        geometry::Frame<LocalTag, LocalTag::tag, /*frame_is_inertial=*/false>;
+    auto const Δplanetarium_rotation =
```
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: I'd be happier to have a type here.

File: ksp_plugin/plugin.cpp [hunk 2]
```diff
@@ -418,9 +418,8 @@ void Plugin::InsertOrKeepLoadedPart(
...
+        Exp(Δt * angular_velocity_of_world_).Forget();
     BodyCentredNonRotatingDynamicFrame<Barycentric, MainBodyCentred> const
```
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: I wonder if this computation could be factored with `BarycentricToWorld`.  Put a TODO to investigate.

File: ksp_plugin/plugin.cpp (hunk 1 above)
Comment (eggrobin) [reply ksp_plugin/plugin.cpp]: Done.

File: ksp_plugin/plugin.cpp (hunk 2 above)
Comment (eggrobin) [reply ksp_plugin/plugin.cpp]: Done.

Comment (eggrobin) [reply ksp_plugin/plugin.cpp]: Added an enum.

**Pull request merged**